test_helpers.py

```
@@ -109,6 +109,25 @@ def profile():
 
 # --- RCE (Remote Command Execution) ---
 @app.route('/exec', methods=['GET', 'POST'])
+def login():
```

Ruff (0.12.2)

Redefinition of unused `login` from line 68

(F811)